src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -5126,7 +5126,7 @@ private static bool IsInDscContext(ExpressionAst expression)
             return Ast.GetAncestorAst<ConfigurationDefinitionAst>(expression) != null;
         }
 
-        private static void CompleteMemberByInferredType(CompletionContext context, IEnumerable<PSTypeName> inferredTypes, List<CompletionResult> results, string memberName, Func<object, bool> filter, bool isStatic)
+        internal static void CompleteMemberByInferredType(TypeInferenceContext context, IEnumerable<PSTypeName> inferredTypes, List<CompletionResult> results, string memberName, Func<object, bool> filter, bool isStatic)
```

Comment:
Do we know that CompletionContext.Helper is the same thing has TypeInferenceContext.Helper?

Reply:
Only by reading the source. But it was me who refactored the helper into a shared class when I wrote the TypeInferenceContext class.